config/initializers/new_framework_defaults_5_1.rb

```diff
@@ -0,0 +1,14 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 5.1 upgrade.
+#
+# Once upgraded flip defaults one by one to migrate to the new default.
+#
+# Read the Guide for Upgrading Ruby on Rails for more info on each option.
+
+# Make `form_with` generate non-remote forms.
+Rails.application.config.action_view.form_with_generates_remote_forms = false
```

Comment:
Interesting decision by the rails team here to move away from synchronous forms by default. We'll want to check with our `rails_bootstrap_forms` library to make sure it starts taking advantage of this change.

Reply:
https://github.com/bootstrap-ruby/rails-bootstrap-forms/pull/346 is the relevant PR but we won't have to worry about implementing for awhile.